src/components/WorkoutCard/WorkoutCard.js

```diff
@@ -64,6 +65,11 @@ export default function WorkoutCard(props) {
           </CooldownSection>
         </div>
       </Wrapper>
+
+      <SelectButton as={NavLink} to={{ pathname: '/music', workout: workout }}>
```

Comment:
I am not sure how this routing works?

Reply:
As far as I understood, this is the object I am taking to the new location. I read this and got some help from the others. https://reactrouter.com/web/api/location

Reply:
Ah! Thanks :)